ui_framework/src/components/form/range/_range.scss

```diff
@@ -0,0 +1,83 @@
+// The following code is inspired by...
+
+// Github: https://github.com/darlanrod/input-range-sass
+// Author: Darlan Rod https://github.com/darlanrod
+// Version 1.4.1
+// MIT License
+
+// It has been modified to fit the styling patterns of Kibana and
+// to be more easily maintained / themable going forward.
```

Comment:
This credit looks good to me, but I think "themable" is spelled "themeable", based on a quick Google search.